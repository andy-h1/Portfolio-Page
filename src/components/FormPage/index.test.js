import React from 'react';
import { render } from '@testing-library/react';
import { FormPage } from '.';

describe('Form Component', () => {
  it('should render', () => {
    const { container } = render(<FormPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
