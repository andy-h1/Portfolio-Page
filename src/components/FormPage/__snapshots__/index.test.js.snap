// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render 1`] = `
<div
  class="sc-fujyUd kVvZcf"
>
  <h2
    class="sc-pNWxx jatnQK"
  >
    Sign Up Form
  </h2>
  <form
    class="sc-jrsJCI ibREEP"
    novalidate=""
  >
    <input
      class="sc-kEqYlL ttapo"
      id="username"
      name="username"
      placeholder="Username"
      required=""
      type="text"
      value=""
    />
    <input
      class="sc-kEqYlL ttapo"
      id="email"
      name="email"
      placeholder="Email"
      required=""
      type="email"
      value=""
    />
    <input
      class="sc-kEqYlL ttapo"
      id="age"
      name="age"
      placeholder="Age"
      type="number"
      value=""
    />
    <div
      class="sc-crzoUp ihsEbs"
    >
      <input
        class="sc-kEqYlL ttapo"
        id="password"
        name="password"
        placeholder="Password"
        required=""
        type="password"
        value=""
      />
      <button
        class="sc-bqGHjH hHSCJX"
        type="button"
      />
    </div>
    <button
      class="sc-iqAbSa gmsiQs"
      type="submit"
    >
      Register
    </button>
  </form>
  <div
    class="sc-dIsAE bSBGZL"
  />
</div>
`;
